### DISCUSSION TITLE
GOPRO 13 - HOW TO enable GPSU metadata (UTC time and data from GPS)

### DISCUSSION THREAD

**Discussion by MisterGREEN78** [gopro/labs#1175] [Q&A]
Hello everyone,
 
I have a problem with an application for car chronometer, this one allows you to integrate a gopro video but the synchronization is based on the GPSU metadata.

I compiled gpmf-parser to extract metadata from my video and GPSU metadata not exist in gopro 13 file.

I checked in video of previous gopro like version 8 and metadata exist.

How I can enable this GPSU metadata please ?

Thanks in advance,
Anthony
![Screenshot 2024-11-16 105648](https://github.com/user-attachments/assets/2eac0261-22b7-414c-b0d2-08e9f097359c)

**Comment by dnewman-gpsw** [comment]
This is GPMF question, not Labs. 

GPSU is obsolete, as time was not included within the GPS5 payload.  Time is included with GPS9, the standard GPS on HERO13 (and HERO11.)  This upgrade made time more precise.

**Reply by MisterGREEN78** [reply]
Thanks a lot for your answer, I understood now 👌
